src/Cafe/CMakeLists.txt

```diff
@@ -485,7 +485,10 @@ endif()
 
 set_property(TARGET CemuCafe PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 
-target_include_directories(CemuCafe PUBLIC "../")
+target_include_directories(CemuCafe PUBLIC 
+  "../" 
+  ${LIBUSB_INCLUDE_DIR}
+)
```

Comment:
Please avoid manually adding a list of include dirs and libraries like these, and use imported targets instead. I'm not sure how to patch vcpkg dependencies to add imported targets when they are not already provided, but you could always submit a patch to <https://github.com/microsoft/vcpkg/tree/master/ports/libusb>.

Reply:
I was just going off of the usage for the libusb vcpkg, not sure if I would actually know what needs to change to submit a package there either but I will look in to it